### PR TITLE
chore(core): tracking events for doc properties

### PR DESCRIPTION
**Pull request by pengx17**
fix AF-1565

### CODE REVIEW

**Comment by pengx17** [comment]
* **#8651** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/8651?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `canary`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @pengx17 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Comment by pengx17** [comment]
### Merge activity

* **Nov 1, 3:07 AM EDT**: [Graphite](https://app.graphite.dev/github/pr/toeverything/AFFiNE/8651) disabled "merge when ready" on this PR due to: a merge conflict with the target branch; resolve the conflict and try again..
* **Nov 5, 5:43 AM EST**: [Graphite](https://app.graphite.dev/github/pr/toeverything/AFFiNE/8651) rebased this pull request after merging its parent, because this pull request is set to merge when ready.
* **Nov 5, 6:00 AM EST**: A user added this pull request to the [Graphite merge queue](https://app.graphite.dev/merges?org=toeverything&repo=AFFiNE).
* **Nov 5, 6:45 AM EST**: A user  merged this pull request with the [Graphite merge queue](https://app.graphite.dev/merges?org=toeverything&repo=AFFiNE).

File: packages/frontend/core/src/components/blocksuite/block-suite-editor/lit-adaper.tsx [hunk 1]
```diff
@@ -251,6 +257,28 @@ export const BlocksuiteDocEditor = forwardRef<
...
+  const onPropertyChange = useCallback((property: DocCustomPropertyInfo) => {
+    track.doc.inlineDocInfo.property.editProperty({
+      type: property.type,
+    });
+  }, []);
+
+  const onPropertyAdded = useCallback((property: DocCustomPropertyInfo) => {
+    track.doc.inlineDocInfo.property.addProperty({
+      type: property.type,
+      module: 'at menu',
```
Comment (forehalo) [inline packages/frontend/core/src/components/blocksuite/block-suite-editor/lit-adaper.tsx]: make sure this module is predefined in `track/src/events.ts`

Comment (forehalo) [reply packages/frontend/core/src/components/blocksuite/block-suite-editor/lit-adaper.tsx]: avoiding anonymous event metrics, we will have clear reports

**Review by forehalo — APPROVED** [review]

**Pull request merged**